spacy_llm/pipeline/llm.py

```diff
@@ -160,6 +166,7 @@ def __init__(
             binary .spacy files. Docs found in the cache directory won't be reprocessed.
         """
         self._name = name
+        self._task = task
```

Comment:
I'm confused, why is this in this PR?

Reply:
mistake! I added this when I was debugging this change, didn't mean to keep it in here